data/human/marauders.txt

```diff
@@ -779,7 +797,7 @@ ship "Marauder Quicksilver"
 		"weapon capacity" 85
 		"engine capacity" 75
 		"reverse thruster slot" 1
-		"steering slot" 1
+		"steering slot" 2
```

Comment:
```suggestion
		"steering slot" 1
```

I think the "little bit faster" here is sufficiently represented by the increase in engine capacity and the better engine, it doesn't need the slot here. (which would also translate to the engine variant having 3, and the other variants having 2 as well)